lib/sourceror.ex

```diff
@@ -906,4 +909,13 @@ defmodule Sourceror do
     |> Enum.chunk_every(2)
     |> Enum.map(&Enum.join/1)
   end
+
+  defp locals_without_parens() do
+    if Version.match?(System.version(), ">= 1.13.0") do
+      {_formatter, formatter_opts} = Mix.Tasks.Format.formatter_for_file("elixir.ex")
```

Comment:
where is `"elixir.ex"` coming from? `formatter_for_file` runs relative to the cwd, so this relies on the calling project to actually have an `elixir.ex` right?

Reply:
`formatter_for_file` does not expect an existing file. You can test this is any `mix` project:
```elixir
iex(1)> Mix.Tasks.Format.formatter_for_file("bar/foo.ex")
{#Function<13.100194628/1 in Mix.Tasks.Format.find_formatter_for_file/2>,
 [
   sigils: [],
   plugins: [],
   inputs: ["{mix,.formatter}.exs",
    "{config,lib,test,script,bench}/**/*.{ex,exs}"],
   import_deps: [:prove, :benchee_dsl],
   locals_without_parens: [
     prove: 1,
     prove: 2,
     after_each: 0,
     after_each: 1,
     after_scenario: 0,
     after_scenario: 1,
     before_each: 0,
     before_each: 1,
     before_scenario: 0,
     before_scenario: 1,
     config: 1,
     formatter: 2,
     inputs: 1,
     job: 1,
     job: 2,
     job: 3,
     jobs: 1
   ]
 ]}
```
If we just want the `opts` we can also call: `Mix.Tasks.Format.formatter_for_file("bar/foo.ex")`. The options are just returned for reflection purposes. The use case of this function is to get the formatter function.
```elixir
iex(1)> {formatter, _opts} = Mix.Tasks.Format.formatter_for_file("bla.ex")
iex(2)> formatter.("defmodule   Foo  do\n\n\n  end") |> IO.puts()
defmodule Foo do
end

:ok
iex(3)> {formatter, _opts} = Mix.Tasks.Format.formatter_for_file("bla.json")
iex(4)> formatter.("defmodule   Foo  do\n\n\n  end") |> IO.puts()
defmodule   Foo  do


  end
:ok
```

Reply:
ooo thanks nick, this was blowing my mind otherwise - was like "how is this passing tests and not going kaboom o.O"